libnetdata/socket/security.c

```diff
@@ -116,10 +116,10 @@ static SSL_CTX * security_initialize_openssl_server(){
     return ctx;
 }
 
-void security_start_ssl(int type){
-    if ( !type){
+void security_start_ssl(int type) {
+    if (!type) {
         struct stat statbuf;
-        if ( (stat(security_key,&statbuf)) || (stat(security_cert,&statbuf)) ){
+        if ((stat(security_key,&statbuf)) || (stat(security_cert,&statbuf))) {
```

Comment:
looks like extra `()`

```c
(stat(security_key,&statbuf) || stat(security_cert,&statbuf))
```

Reply:
I decided to use the extra () to isolate the results, I imagined that this was more simple to read, but I am fixing with the next pull.

Reply:
The last pull fixed this!